spec/vehicle_spec.rb

```diff
@@ -33,4 +37,31 @@
       expect(@camaro.electric_vehicle?).to eq(false)
     end
   end
+
+  describe '#set registration date' do
+    it 'starts at nil when not in hash' do
+      expect(@cruz.registration_date).to eq(nil)
+    end
+
+    it 'puts todays date as registration date' do
+      expect(@cruz.set_registration_date).to eq(Date.today)
+      expect(@bolt.set_registration_date).to eq(Date.today)
+      expect(@camaro.set_registration_date).to eq(Date.today)
+    end
+  end
+
+  describe '#give plate' do
+    it 'has a plate type' do
+      expect(@cruz.plate_type).to eq(nil)
+
+      @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice, plate_type: :regular} )
+      @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev, plate_type: :ev} )
+      @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice, plate_type: :antique} )
+
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@camaro.plate_type).to eq(:antique)
+    end
```

Comment:
Good job writing tests for the additional methods you created in the Vehicle class.